doc/source/tutorials/pidloops.rst

```diff
@@ -309,7 +309,7 @@ The script we'll use to tune the highly overpowered rocket shown will launch the
 
     // feedback based on atmospheric efficiency
     LOCK surfspeed TO SHIP:VELOCITY:SURFACE:MAG.
-    LOCK atmoeff TO surfspeed / SHIP:TERMVELOCITY.
+    LOCK atmoeff TO surfspeed / SHIP:TERMVELOCITY.  //Using deprecated TERMVELOCITY syntax
```

Comment:
Thanks!  I can look into the tutorial; is there a timeline for when you want this done by?